test/e2e/operator/nmstate_install_test.go

```diff
@@ -130,6 +131,40 @@ var _ = Describe("NMState operator", func() {
 			})
 		})
 	})
+	Context("when cluser-reader exists", func() {
+		const (
+			clusterReaderRoleName = "cluster-reader"
+			testUserNamespace     = "default"
```

Comment:
maybe better on the nmstate NS, so once it is deleted it will clean tests leftovers if any